gunrock/app/kcore/kcore_enactor.cuh

```diff
@@ -0,0 +1,578 @@
+//-----------------------------------------------------------------
+// Gunrock -- Fast and Efficient GPU Graph Library
+// ----------------------------------------------------------------
+// This source code is distributed under the terms of LICENSE.TXT
+// in the root directory of this source distribution.
+// ----------------------------------------------------------------
+
+/**
+ * @file
+ * kcore_enactor.cuh
+ *
+ * @brief k-core Problem Enactor
+ */
+
+#pragma once
+
+#include <gunrock/app/enactor_base.cuh>
+#include <gunrock/app/enactor_iteration.cuh>
+#include <gunrock/app/enactor_loop.cuh>
+#include <gunrock/oprtr/oprtr.cuh>
+
+#include <gunrock/app/kcore/kcore_problem.cuh>
+
+//#define PRINT_DEBUG(FLAG, ID, K) (FLAG && (ID == 3366) && (K <= 129))
+#define PRINT_DEBUG(FLAG, ID, K) false
```

Comment:
Should this be hard-coded to false?

Reply:
It's just for print debugging when/if I have a reproducible bug, so basically I just set it to off to avoid printing a bunch of stuff. I never removed these because I'm still not sure if I fixed the bug, but I can do that if you think I should.